### PR TITLE
Testing recipe to grocery

### LINKED ISSUE #35: recipe to grocery list use case testing + documentation

### PR DESCRIPTION
**Pull request by hhpfiona**
created tests for recipe to grocery, with documentation as well. please let me know if changes are needed.

### CODE REVIEW

**Comment by parkhangorodsky** [comment]
link issue to this PR

File: .idea/encodings.xml
Comment (parkhangorodsky) [inline .idea/encodings.xml:1]: .idea files don't need to be pushed

File: .idea/misc.xml
Comment (parkhangorodsky) [inline .idea/misc.xml:1]: .idea files don't need to be pushed

File: src/main/java/entity/Recipe.java [hunk 1]
```diff
@@ -1,5 +1,6 @@
 package entity;
 
+import java. util. ArrayList;
```
Comment (parkhangorodsky) [inline src/main/java/entity/Recipe.java:3]: is this line right? there are some extra spaces on my end

File: target/classes/app/Config.class
Comment (parkhangorodsky) [inline target/classes/app/Config.class:1]: don't need to push target files for the future

**Review by parkhangorodsky — APPROVED** [review]
lots of repeated commits as in Fiona's other PR - might be good to avoid this in the future (for files that you aren't adding onto) since it adds a lot to (re)read.

Otherwise, if this is only documentation/tests, then this is good

Edit: this could be fixed maybe by branching off of your current branch, then merging back into that - instead of both into main

**Review by RqedK — APPROVED** [review]

**Review by seonghyunban — APPROVED** [review]
Again please make a separate branch for testing, debugging, or miscellaneous modifications.

**Pull request merged**